src/components/molecules/NavSearchBar/NavSearchResult.scss

```diff
@@ -0,0 +1,38 @@
+@import "scss/constants.scss";
```

Comment:
[note] I haven't deeply reviewed all of the specific styles in this file, more just the overall structure/etc of it.